### ISSUE TITLE
Live Sync stops working after +-48 hours until Bitwarden instance is restarted.

### ISSUE THREAD

**Issue by karelvanhecke** [bitwarden/server#501]
Live sync stops working after the instance has been running for around 48 hours.
I was able to reproduce the issue consistently with a clean, default Bitwarden instance. 

All containers seem to be running properly and nothing in the logs indicates what the issue is.

Bitwarden deployment:
Docker-CE on Centos 7

Tested with all clients, also tested without the mobile client to rule out push notification issues.

**Comment by kspearrin** [comment]
What version are you using? There was an edge case that could cause the notifications worker to crash. This has been fixed in 1.30.4,

**Comment by karelvanhecke** [comment]
I was still on 1.30.3. Should this crash be traceable in Bitwarden's logs?

**Comment by kspearrin** [comment]
It may be. Have a look to see if there are any logs under `./bwdata/logs/notifications`. The notifications container is the one responsible for sending the live sync messages to clients.

**Comment by karelvanhecke** [comment]
Seems that there aren't any logs written by the notifications container. Anyway, I've updated to 1.30.4, I'll report back if the issue remains.

**Comment by karelvanhecke** [comment]
@kspearrin Issue still persists. Bitwarden instance has been running for 46 hours since I updated to 1.30.4 and live sync no longer works. Nothing is logged in `bwdata/logs/notifications`.

I did notice something in the nginx access log though.

When a connection for live sync notifications is established, you'll see the following requests in your logs:

```
POST /notifications/hub/negotiate HTTP/2.0
GET /notifications/hub?<ids and access tokens> HTTP/1.1
```

When live sync no longer works, the `POST /notifications/hub/negotiate HTTP/2.0` request is still sent, but the second request that actually establishes the WebSocket connection (I assume) is no longer sent.

Note that this only occurs when the instance has been running for 46-48 hours. If I schedule a daily restart, this issue never occurs.
